ChainCfg:
  samplerate: 512
  order: 2
  cutoff_low: 2
  cutoff_high: 40
  lap_path: "/home/curtaz/Neurorobotics/neurorobotics-ws/src/rosneuro_dl/config/laplacian16.mat" 